baystation12.dme

```diff
@@ -3634,10 +3634,10 @@
 #include "interface\skin.dmf"
 #include "maps\_map_include.dm"
 #include "maps\_maps.dm"
-#include "maps\away\icarus\icarus_areas.dm"
 #include "maps\away\geminus_city\road_and_pavements.dm"
 #include "maps\away\geminus_city\signs.dm"
 #include "maps\away\geminus_city\streetobjects.dm"
+#include "maps\away\icarus\icarus_areas.dm"

```

Comment:
this shouldn't be part of this pr

separately, icarus_areas needs to be moved to icarus.dm but @CPU-Blanc may have already done that in an upcoming pr

Reply:
`icarus_areas` is already included in `icarus.dm`, without any PRs or anything like that.

Reply:
definitely should be removed from the dme entirely in that case, if you want to just do that here @scrdest

Reply:
Oh, sure. IDK what's up with that, it's the DME acting weird again and it's not even that the IDE was open this time

Reply:
Should be sorted